### DISCUSSION TITLE
Rounded corners question

### DISCUSSION THREAD

**Discussion by SamKotBrown** [ImaginativeShohag/Why-Not-Image-Carousel#56] [Q&A] [answered]
Hello.

I just started using this amazing image viewer today but i have an issue.

I'm trying to put rounded corners on the image carousel container but i don't know how and i can't find any tutorial that explains how to change them. Also i don't know how to edit the style of the Circle indicator.
Could you help me a little bit with it ?

**Comment by ImaginativeShohag** [comment, marked as answer]
You can make round corners in many ways. See the samples from [here](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/sample). You can wrap your view using `MaterialCardView` and change the `app:cardCornerRadius` value to round the corners.

### About `CircleIndicator`:

Yes you can customize `CircleIndicator2`. You can add your custom `CircleIndicator2` by using `setIndicator()` method like this:

```kotlin
val customIndicator: CircleIndicator2 = findViewById(R.id.custom_indicator)
carousel.setIndicator(customIndicator)
```

Or you can `getIndicator()` and customize the existing indicator programmatically.
